### DISCUSSION TITLE
Change top level of albums

### DISCUSSION THREAD

**Discussion by Revorge** [photoview/photoview#362] [Ideas]
Hi,

I am currently testing photoview and thus far I am really impressed :)

My photos are mounted to /photos as per the docker instructions. Was wondering if there is a way to make that folder the "top level" / root folder, so that clicking Albums in the menu shows the content (subfolders) of /photos instead of having it as a seperate item. I think this makes sense when only one photo path is configured. Sorry if I somehow missed an obvious way of doing this ;)

![temp-pw](https://user-images.githubusercontent.com/32901816/117227785-e981a280-ae17-11eb-9b46-8d9924cc9581.JPG)

**Comment by viktorstrate** [comment]
Hi @Revorge,

Yes, it might be worth adding an edge case for users with only one root path, and show the content of the directory directly.

**Reply by Revorge** [reply]
That would be perfect. Especially since I am personally more likely to mount additional folders within that path, but I have other users that will probably prefer multiple "root paths".

Thanks for the swift reply :)

**Reply by viktorstrate** [reply]
Hey @Revorge, thank you for the donation, I really appreciate it. ❤️
It's nice to see a fellow Dane here ☺️

**Reply by Revorge** [reply]
I have been looking for a photo gallery that suited my needs better than the common solutions for "ages". So supporting you was an easy decision. 

I live pretty close to SDU actually :)
